src/common/dao/repository_test.go

```diff
@@ -222,30 +222,50 @@ func TestGetRepositoriesByProject(t *testing.T) {
 }
 
 func TestGetAllRepositories(t *testing.T) {
+	require := require.New(t)
+
 	var reposBefore []*models.RepoRecord
-	reposBefore, err := GetAllRepositories()
-	if err != nil {
-		t.Fatalf("error occurred while getting all registories. %v", err)
+	reposBefore, err1 := GetAllRepositories()
+	if err1 != nil {
+		t.Fatalf("error occurred while getting all registories. %v", err1)
 		return
 	}
 	allBefore := len(reposBefore)
 
+	project1 := models.Project{
+		OwnerID: 1,
+		Name:    "projectRepo",
+		Public:  0,
+	}
+	var err2 error
+	project1.ProjectID, err2 = AddProject(project1)
+	require.NoError(err2)
+
 	for i := 0; i < 1200; i++ {
 		end := strconv.Itoa(i)
 		repoRecord := models.RepoRecord{
 			Name:      "test" + end,
-			ProjectID: 1,
+			ProjectID: project1.ProjectID,
 		}
 		if err := AddRepository(repoRecord); err != nil {
 			t.Fatalf("Error happens when adding the repository: %v", err)
 			return
 		}
 	}
+	defer func() {
```

Comment:
Use this to clean data:
https://github.com/vmware/harbor/blob/master/src/common/dao/base.go#L106